deployment/CP05MOAS-GL379_Deploy.csv

```diff
@@ -35,9 +35,15 @@ TS20190205,,TS20190516,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,2019
 TS20190205,,TS20190516,,CP05MOAS-GL379-03-CTDGVM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09363,39.9166,-70.36972222,,1000,1000,SS-1
 TS20190205,,TS20190516,,CP05MOAS-GL379-04-DOSTAM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9166,-70.36972222,,1000,1000,SS-1
 TS20190205,,TS20190516,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
-WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
-WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1 Will change UID to CGINS-ADCPAM-00548
-WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
-WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
-WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
-WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-18T23:14:29,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1 Will change UID to CGINS-ADCPAM-00548
+WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1

```

Comment:
Looks good, but should they be a recovery cruise #?

Reply:
looks good for P13 deploy; there should be a recover cruise # but it won't 'break' anything if we don't have it in there now.  Could updated after P13 deployments are complete...